library/core/functions.render.php

```diff
@@ -633,6 +633,12 @@ function cssClass($row, $inList = true) {
             }
         }
 
+        if (array_key_exists('imageSource', $row)){
```

Comment:
```suggestion
        if (array_key_exists('imageSource', $row)) {
```

This is what is causing your PHP lint error